feature-libs/organization/administration/components/unit/links/users/list/unit-user-list.component.ts

```diff
@@ -31,5 +32,10 @@ export class UnitUserListComponent {
     ? this.currentUnitService.item$
     : of({ active: true });
 
-  constructor(protected currentUnitService: CurrentUnitService) {}
+  isUpdatingUserAllowed = this.b2bUserService.isUpdatingUserAllowed();
+
+  constructor(
+    protected currentUnitService: CurrentUnitService,
+    protected b2bUserService: B2BUserService
```

Comment:
I think here we have the same case. Please take a look at my comment above.

Reply:
done